assets/test/specs/form.spec.js

```diff
@@ -32,6 +40,8 @@ describe('Form Validation', function () {
     jasmine.getFixtures().fixturesPath = 'base/specs/fixtures/';
     loadFixtures('form-fixture.html');
     form = require('../../javascripts/validation/form.js');
+    toggleDynamic = require('../../javascripts/modules/toggleDynamicFormFields.js');
+    document.body.classList.add("js")
```

Comment:
When it's on the body it borks other tests because it isn't being removed between runs of suites. Fixtures are destroyed with each run so it isolates you tests.